### PR TITLE
init spring boot app

### PR DESCRIPTION
**Pull request by rkondratowicz**
using https://github.com/hmcts/spring-boot-template

all changes were applied after first commit

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Contribution guidelines
+
+We're happy to accept 3rd-party contributions. Please make sure you read this document before you do any work though,
+as we have some expectations related to the content and quality of change sets.
+
+## What you should know about this application
+
+This project is a template Spring Boot application. It aims to speed up the creation of new Spring APIs in HMCTS
```
Comment (lgonczar) [inline .github/CONTRIBUTING.md]: This should be changed, as well.

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+server:
+  # TODO: change to your application's port number
+  port: 4550
```
Comment (lgonczar) [inline src/main/resources/application.yaml]: should be 8484

File: src/test/java/uk/gov/hmcts/reform/jobscheduler/ApplicationTests.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class ApplicationTests {
+
+    @Test
+    public void sample_test() {
```
Comment (lgonczar) [inline src/test/java/uk/gov/hmcts/reform/jobscheduler/ApplicationTests.java]: The initial test here was a valid test. This example should be deleted.

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# Contribution guidelines
+
+We're happy to accept 3rd-party contributions. Please make sure you read this document before you do any work though,
+as we have some expectations related to the content and quality of change sets.
+
```
Comment (lgonczar) [inline .github/CONTRIBUTING.md:5]: We should probably have a very short description about what this application is for

Comment (rkondratowicz) [reply .github/CONTRIBUTING.md:5]: that's what readme is for?

File: src/test/java/uk/gov/hmcts/reform/jobscheduler/ApplicationTests.java (hunk 3 above)
Comment (lgonczar) [reply src/test/java/uk/gov/hmcts/reform/jobscheduler/ApplicationTests.java]: Sorry, my bad - I thought we had a different test here (loading the context)

File: .github/CONTRIBUTING.md (hunk 4 above)
Comment (lgonczar) [reply .github/CONTRIBUTING.md:5]: fair enough

**Review by lgonczar — APPROVED** [review]

**Review by doncem — COMMENTED** [review]
can we have up-to-date versions for init commit? probably should do on spring boot first :)

**Pull request merged**